### PR TITLE
[13.0][MIG] account_partner_reconcile

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: account_partner_reconcile/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
-# Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
-#        (http://www.eficent.com)
+# Copyright 2017-19 ForgeFlow S.L. (http://www.forgeflow.com)
```
Comment (LoisRForgeFlow) [inline account_partner_reconcile/__manifest__.py]: We are in 2020 :wink:

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**